README-ID.md

```diff
@@ -53,14 +53,14 @@ Jika Anda ada pertanyaan atau kommen, silahkan membuat issue baru.
-- [Contributing to Open Source: A Live Walkthrough from Beginning to End](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - This walkthrough of an open source contribution covers everything from picking a suitable project, working on an issue, to getting the PR merged in.
-- ["How to Contribute to Open Source Project by" Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - They are focusing on the nitty-gritty of contributing a pull request (PR) to someone else’s project on GitHub.
+- ["How to Contribute to an Open Source Project on GitHub" by Egghead.io](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - Panduan video bertahap tentang cara mulai berkontribusi pada proyek Open Source di GitHub.
+- [Contributing to Open Source: A Live Walkthrough from Beginning to End](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - Panduan tentang kontribusi open source ini mencakup segala hal mulai dari memilih proyek yang sesuai, mengerjakan suatu issue, hingga menggabungkan PR.
+- ["How to Contribute to Open Source Project by" Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - Mereka berfokus pada intisari kontribusi pull request (PR) ke proyek orang lain di GitHub.
 - ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - Artikel ini membahas sumber untuk berkontribusi ke Open Source bagi para pemula berdasarkan minat bahasa favorit mereka.
 - ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub sekarang membantu Anda menemukan issue pertama yang baik untuk mulai berkontribusi ke Open Source.
-- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - This comprehensive article is directed towards businesses (but still useful for individual contributors) where it talks about why, how, and what open-source projects to contribute to.
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - Artikel komprehensif ini ditujukan untuk perusahaan (tetapi masih berguna untuk kontributor individu) yang membahas tentang mengapa, bagaimana, dan proyek Open Source apa yang dapat dikontribusi.
 - ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -
-Lets Git started in the world of opensource, starting in the opensource playground. Especially designed for education and practical experience purposes.
+Mari mulai menggunakan Git dalam dunia Open Source, mulai dari  Opensource Playground. Dirancang khusus untuk tujuan edukasi and pengalaman praktis.
 
 
 ## Pencarian GitHub secara Langsung
```

Comment:
The title does not describe the content (maybe wordings issue?)

Reply:
I mean it's difficult to translate it directly since it plays with something only exists in the English language and that I think is the closest thing, any suggestion is welcome